lib/desktop-notifications.ts

```diff
@@ -46,6 +35,25 @@ function closeNotification(id: string) {
   return nativeModule.closeNotification(id)
 }
 
+type DesktopNotificationEvent = 'click'
+
+const shownNotifications = new LRUCache<string, DesktopNotification>({
+  max: 50,
```

Comment:
@niik I remember you mentioned a much higher number (300? 500 maybe?). I think 50 should be enough, but if you don't think it's a problem, I'm happy to bump it to any of those values.

Reply:
Right, I think I said something along the lines of an "arbitrary high number, just not an unbounded number". Storing 50 of these or even 5000 isn't going to be a big deal memory-wise, we just don't want to end up in an edge-case where some long running instance coupled with either a bug from us or an unanticipated use pattern generating 100k of these.

I just counted (yup, like manually counted) the number of unread slack notifications I had on my mac as of right now and it was 63 😓  So how about we make it 100 to make it @niik-proof with some padding?

```suggestion
  max: 100,
```